uint16 sensor_type
uint16[] enu_pos
uint16 heading_ang
uint16 ob_case